noether_conversions/CMakeLists.txt

```diff
@@ -40,6 +43,7 @@ catkin_package(
     geometry_msgs
     noether_msgs
   DEPENDS
+    tool_path_planner
     EIGEN3
```

Comment:
Eigen doesn't define `EIGEN3_INCLUDE_DIRS` or `EIGEN_LIBRARIES` variables, so it should be removed from the DEPENDS section to fix build warnings

Reply:
Shouldn't this be listed as `Eigen3` which is what find_package calls?  I also think you still need to list Eigen here if Eigen is used in any of the headers. I believe this list is used to generate the package CMake config file where it calls find_package on each of the names listed here. The reason for the warning is that catkin was pre Targets so it always assumed there should be these variables for every package listed. Though, this is not the case anymore because libraries are starting to export targets and will eventually stop creating these types of variables.